### ISSUE TITLE
16.5 problems on Fire TV2

### ISSUE THREAD

**Issue by adpine58** [koying/SPMC#606]
I have been enjoying a VERY stable SPMC on my Fire TV 2 until i upgraded to 16.5.  Now going back and forth within lists in Exodus are very slow, almost making me think its frozen sometimes. I have also noticed that every show i start to stream, the audio delay is 5 seconds or so.  Last night i was using Release Hub to watch a show.  when i stopped the show it kept restarting.  i exited out of SPMC, went back in and the movie started again.  i thought i was going to have to just shut the Fire tv off but i figured out a way.

in short its no longer as pleasant of an experience.  i have for now gone back to 16.4.2 which still seems fine

was 16.5 a major upgrade? should i be concerned about the Fire Tv device being supported on this newer version. i am happy to wait till the new version settles down if thats what i need to do

oh, and thank you for all you do to provide this software.  i dont want to make it sound like i am just complaining without showing appreciation for all that it does

**Comment by JFG90** [comment]
Really odd as we have 16.5.3 on our fire tv and it plays things on exodus perfectly... Uninstall and reinstall from playstore to revert back to 16.4.2

**Comment by adpine58** [comment]
It was really the Release Hub issue that was the worst.  When i stopped the show it would go back to the previous list and then just start the show up again as if i had pressed another button.  i exited SPMC by hitting the home button and the show even restarted out of the Amazon fire tv home screen on its own.  i have never seen this type of thing before

**Comment by christoophat** [comment]
Sounds more of an Exodus issue than an SPMC one.

**Comment by adpine58** [comment]
Well i have tried again to update to the newest version of 16.5.  This is the 3rd version i have tried and they all have the same problems on my Fire TV 2nd gen.  I use Exodus and Release Hub and Stream all the sources.  These all work perfectly on 16.4.2 but when i try on 16.5 there are 2 main problems i get.  All of these addons have 1 of the problems and that is slowness.  Going from list to list takes around 5 seconds each time, even going back to a previous list.  on v16.4.2 its almost instantaneous.  this might not seem like a big deal but it really does affect the usefulness. the 2nd problem and worse in my opinion is in Release Hub.  when stopping a show i am watching, when i go back to the menu the show restarts automatically.  this loop is almost impossible to break without shutting off the Fire TV altogether

Now i know its easier to blame the addons and maybe they are at fault.  Is there something in 16.5 that could cause Addons to start failing?  do they need to make changes to support the new version of SPMC?

I am beginning to wonder if the Fire Tv will continue to be supported by SPMC as it moves forward....

for the time being i have gone back to 16.4.2 and again to seems a perfect working version for me

Let me just say that i really appreciate all the work that goes into this software.....thank  you

**Comment by christoophat** [comment]
Q: Does SPMC  work normally other than when you use those 3rd party addons? If the answer is yes, then you should probably go ask the addon devs for help, since they are the ones writing code to work w/ SPMC, not the other way around, no?
